ocf_datapipes/select/offset_t0.py

```diff
@@ -0,0 +1,21 @@
+from typing import Union
+
+import numpy as np
+from torchdata.datapipes import functional_datapipe
+from torchdata.datapipes.iter import IterDataPipe
+
+
+@functional_datapipe("offset_t0")
+class OffsetT0IterDataPipe(IterDataPipe):
+    def __init__(
+        self,
+        source_dp: IterDataPipe,
+        max_t0_offset_minutes: Union[float, int],
+        min_t0_offset_minutes: Union[float, int] = 0.0,
+    ):
+        self.source_dp = source_dp
+        self.max_t0_offset_minutes = max_t0_offset_minutes
+        self.min_t0_offset_minutes = min_t0_offset_minutes
```

Comment:
what will this do? randomly take between the max and min?

Reply:
I was thinking that would be the idea? So that each training example can have the most randomness when training models, and we can give it for satellite like t0_min=30min, t0_max=90min to simulate the delay in data availablility.